src/game/city.go

```diff
@@ -262,3 +263,36 @@ func upgradeBuilding(response http.ResponseWriter, request *http.Request) {
 
 	status = true
 }
+
+func updateName(response http.ResponseWriter, request *http.Request) {
+	fmt.Println("Received request to /cities/updateName")
+	vars := mux.Vars(request)
+	sessionId := vars["session_id"]
+	status := false
+
```

Comment:
Need to take into account changing name of other towns that you own, so you should also be taking in a query param of cityName. Check the other functions to see how this is done